index.html

```diff
@@ -915,12 +915,182 @@ <h4>Get a Specific Presentation</h4>
+state of the exchange as it progresses and to obtain information associated
+with the exchange that the workflow service has stored. Querying the exchange
+in this way requires additional authorization that the coordinator is expected
+to have and that the exchange client is not.
+      </p>
+      <p>
+How the exchange URL is transmitted from a coordinator to an exchange client is
+out of scope for this specification. Known mechanisms for sharing the exchange
+URL with the client include using the Credential Handler API (aka CHAPI), a QR
+code, or a universal link.
+      </p>
+      <p>
+VC API exchanges can be executed using the VC API exchange protocol but are
+also designed to be executable with additional protocols. For example, an
+exchange could potentially also be executable via protocols such as OID4VCI,
+OID4VP, and DIDComm. Which protocols are supported depend on the workflow
+service implementation, the complexity of the workflow the exchange is based
+off of, and the options provided by the coordinator when the exchange was
+created.
+      </p>
+      <p>
+The exchange client is expected to initiate the exchange using a protocol that
+is compatible with how the client received the exchange URL. For example, the
+exchange URL could have been provided over CHAPI with a protocol identifier
+indicating that the VC API protocol ought to be used. Alternatively, the
+exchange URL could be included as the "credential_issuer" in an OID4VCI
+credential offer, or as the "client_id" of an OID4VP authorization request,
+indicating that OID4VCI or OID4VP, respectively, ought to be used. This section
+focuses on how an exchange client uses VC API to interact with the exchange,
+but see appendix X to see how to combine VC API exchanges with other protocols
+such as OID4VCI, OID4VP, and DIDComm.
+      </p>
+      <p>
+Exchanges that are executed using the VC API protocol involve messages sent as
+request and response bodies over HTTP. Each message consists of a simple JSON
+object that includes zero or more of the following properties and values:
+      </p>
+      <ul class="bullet">
+        <li>
+`verifiablePresentationRequest`: A Verifiable Presentation Request. This is
+used by one party in the exchange to request information from the other party.
+        </li>
+        <li>
+`verifiablePresentation`: A Verifiable Presentation. This is used by one party
+to provide information requested by the other party or simply offered by the
+sender.
+        </li>
+        <li>
+`redirectUrl`: A URL that can be used to continue an interaction at another
+location. One use case for this is to redirect the user of an exchange client
+back to a coordinator website after an exchange has completed.
+        </li>
+        <li>
+Any custom properties and values might also be included, but are expected to
+trigger errors in implementations if they are not recognized.
+        </li>
+      </ul>
```

Comment:
This doesn't seem like a bullet item to me. It might be better to move and blend this sentence into the paragraph leading into the bullet list.
```suggestion
      </ul>
      <p>
Custom properties and values might also be included, but are expected to
trigger errors in implementations that do not recognize them.
      </p>
```

Reply:
Yeah, I couldn't decide whether to put this in the bulleted list or not. One reason to do it is so that eyes are drawn to it vs. having to scan a paragraph for it.